wiki/Article_Style_Guide/en.md

```diff
@@ -764,11 +632,52 @@ Some tools for beautifying tables are:
 - [Markdown Table Formatter](http://markdowntable.com/)
```

Comment:
Please, take the time to make sure https works before commenting about it. And no, this site does not support https.